accounts/views.py

```diff
@@ -729,6 +729,39 @@ def attribution(request):
     return render(request, 'accounts/attribution.html', tvars)
 
 
+@login_required
+def download_attribution(request):
```

Comment:
could you also try and write a test for this method? Something that has at least one sound and at least one pack, and then check that both formats give the expected result

Reply:
Should I integrate it into `test_views` or write a separate test file?

Reply:
Thank you, I found that already, and saw that setting `num_packs=1` in the test setup shouldn't disturb the other tests; I'll go that way.

Reply:
Push done with the above issues addressed and a first draft of the tests added.

1. Should I compare the content with the actual object properties instead of the fixed strings, in case the `create_user_and_sounds` ever gets changed?
2. Should I fix the pack name appearing instead of the pack license (and if so, how), or can it be ignored for the tests?
3. Should I add a test for the redirection to the `attribution` view in case of unexpected values for the `dl` parameter?

Reply:
1. answering myself, yes, fixed strings won't cut it. Besides, while the new tests were working via separate test calls, I broke the whole suit; back to the drawing board...

Reply:
Ok, the tests are fixed now.